test/verify/check-storage-smart

```diff
@@ -82,17 +85,27 @@ class TestStorageSmart(storagelib.StorageSmartCase):
         set_smart_dump("INTEL_SSDSA2MH080G1GC--045C8820", "/dev/sda")
         check_smart_info("Disk is OK", "2309 hours", "Interrupted")
 
-        # Aborted self test and has known bad sector
+        # latest libblockdev builds from copr have different behavior when assessing disk as failed
+        is_nightly_scenario = "daily" in getenv("TEST_SCENARIO", "")
+
+        # Aborted self test and has known bad sector (overall assessment is still OK)
         set_smart_dump("ST9160821AS--3.CLH", "/dev/sda")
-        check_smart_info("Disk is failing", "556 hours", "Aborted", bad_sectors="1")
+        if is_nightly_scenario:
+            check_smart_info("Disk is OK", "556 hours", "Aborted", bad_sectors="1")
```

Comment:
I suggest to factorize this, do the version check and set some `overall_status = "..."` string conditionally there, then use the string here and below.